### PR TITLE
Add personal caption to image

### LINKED ISSUE #8: beardofedu Workflow
beardofedu will choose an image, add a caption, and add both to a file.

- [ ] Create a branch
- [ ] Edit the file
- [ ] Commit the changes
- [ ] Create a Pull Request
- [ ] Request a Review
- [ ] Make more changes
- [ ] Get an approval
- [ ] Merge the Pull Request

### PR DESCRIPTION
**Pull request by beardofedu**
Creating a pull request to change my caption. 

@PPatilGitLearn modified this to make it funny 😂  what are your thoughts? 💭 

Closes #8

### CODE REVIEW

**Review by PPatilGitLearn — APPROVED** [review]
Everything seems perfect. Good to go.

File: _posts/0000-01-16-githubteacher.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![evil-raccooon](https://cloud.githubusercontent.com/assets/16547949/25400793/b1104eee-29c1-11e7-89e2-3c99f9ea1941.jpg)
 {: .center}
 
-Caption here
+I really miss Groot.
```
Comment (ravurimca) [inline _posts/0000-01-16-githubteacher.md]: Okay

Comment (PPatilGitLearn) [inline _posts/0000-01-16-githubteacher.md]: Ok. Reviewing Changes

**Comment by PGhosh3** [comment]
:fireworks:

**Review by PPatilGitLearn — APPROVED** [review]
Approved.

**Review by ravurimca — CHANGES_REQUESTED** [review]
Requesting changes for no reason

**Pull request merged**